Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/CustomException.java

```diff
@@ -0,0 +1,11 @@
+package tddmicroexercises.telemetrysystem;
+
+public class CustomException {
```

Comment:
lets rename it to meaningful exception like TelemetryException
and also follow this blog and update the code accordingly